_overviews/jdk-compatibility/overview.md

```diff
@@ -4,70 +4,79 @@ title: JDK Compatibility
 
-This table shows the first Scala release in each series that works with each JVM release.
-
-| JVM version | Minimum Scala versions                                                                        |
+| JDK version | Minimum Scala versions           | Recommended Scala versions                                 |
 |:-----------:|:----------------------------------------------------------------------------------------------|
-| 13, 14      | 2.13.1, 2.12.9, see [below](#jdk-13-and-14-compatibility-notes) for further notes                    |
-| 12          | see [below](#jdk-12-compatibility-notes)                                                      |
-| 11          | 2.13.0, 2.12.4, 2.11.12, 2.10.7 (but also, see [below](#jdk-11-compatibility-notes))          |
-| 8           | 2.13.0, 2.12.0, 2.11.0, 2.10.2                                                                |
-| 7           | 2.11.0, 2.10.0                                                                                |
-| 6           | 2.11.0, 2.10.0                                                                                |
+| 13, 14      | 2.13.2, 2.12.11                  | 2.13.2, 2.12.11                                            |
+| 12          | 2.13.1, 2.12.9                   | 2.13.2, 2.12.11                                            |
+| 11          | 2.13.0, 2.12.4, 2.11.12          | 2.13.2, 2.12.11, 2.11.12                                   |
+| 8           | 2.13.0, 2.12.0, 2.11.0, 2.10.2   | 2.13.2, 2.12.11, 2.11.12, 2.10.7                           |
+| 6, 7        | 2.11.0, 2.10.0                   | 2.11.12, 2.10.7                                            |
+
+Even when a version combination isn't listed as supported, most features may still work.  (But Scala 2.12+ definitely doesn't work at all on JDK 6 or 7.)
+
+In general, Scala works on JDK 11+, including GraalVM, but it might not take advantage of features that were added after JDK 8. See[below](#jdk-11-compatibility-notes).
+
+Lightbend offers [commercial support](https://www.lightbend.com/lightbend-platform-subscription) for Scala. The linked page includes contact information for inquiring about supported and recommended versions.
 
 ## Running versus compiling
 
-We recommend using Java 8 for *compiling* Scala code. Since the JVM is backward compatible, it is usually safe to use a newer JVM to *run* your code compiled by the Scala compiler for older JVM versions. There are notable exceptions with experimental/unsafe features, and the introduction of the module system in Java 9. The Scala compiler does usually need updates to run properly on newer versions of the JVM, so make sure to use the appropriate JVM when compiling your code.
+We usually recommend JDK 8 or 11 for *compiling* Scala code. Since the JVM is usually backward compatible, it is usually safe to use a newer JVM for *running* your code, especially if you are not using JVM features designated "experimental" or "unsafe".
```

Comment:
that's a lot of "usually"s. could we use other words without losing precision?

```suggestion
We generally recommend JDK 8 or 11 for *compiling* Scala code. Since the JVM tends to be backward compatible, it is usually safe to use a newer JVM for *running* your code, especially if you are not using JVM features designated "experimental" or "unsafe".
```